submissions/ivan-chukhalo/side-menu/index.js

```diff
@@ -0,0 +1,97 @@
+  },
+  {
+    name: "Чорнокнижник",
+    description:
+      "Чорнокнижники знищують ослабленого противника, поєднуючи хвороби і темну магію. Перебуваючи під захистом своїх вихованців, чаклуни розбивають ворога на відстані. Фізично слабкі чаклуни що неспроможні носити важку броню, тому підставляють під ворожі удари своїх слуг.",
+  },
+  {
+    name: "Чернець",
+    description:
+      "Незалежно від стилю ченці в бою покладаються в основному на свої кулаки і ноги, а зв'язок з внутрішньою енергією ци надає їм сил для використання особливих здібностей. Вони здатні одночасно зцілювати союзників і завдавати шкоди противникам.",
+  },
+  {
+    name: "Друїд",
+    description:
+      "Друїди можуть підходити до битви по-різному. Вони вільні грати майже будь-яку роль у команді: бути цілителями, танками чи бійцями, але повинні пам'ятати про особливості кожної ролі. Друїд змушений уважно підбирати вигляд до ситуації, оскільки кожен із навичків служить певної мети.",
+  },
+  {
+    name: "Мисливець на демонів",
+    description:
+      "Мисливці на демонів не користуються важкою бронею, натомість покладаючись на швидкість, яка дає їм можливість стрімко наближатися до супротивника і завдавати йому шкоди одноручною зброєю. Однак іллідары не варто забувати про те, щоб використовувати свою спритність і в захисних цілях, забезпечуючи собі сприятливий результат битви.",
+  },
+  {
+    name: "Лицар смерті",
+    description:
+      "Лицарі смерті сходяться з супротивником у ближньому бою, доповнюючи удари клинка темною магією, яка робить ворога вразливим або ранить його безбожною енергією. Вони провокують противників, змушуючи їх боротися віч-на-віч і не підпускаючи їх до слабших союзників. Щоб не дати супротивникові вислизнути, лицарі смерті повинні постійно пам'ятати про силу, яку витягують з рун, і відповідним чином спрямовувати свої атаки.",
+  },
+];
+
+let classesList = document.getElementById("menu");
+let title = document.getElementById("content__title");
+let description = document.getElementById("content__description");
+let classImg = document.getElementById("content__img");
+let linkHome = document.getElementById("content__link_back");
+
+function printClassInformation(event) {
+    let menuItem = event.target.closest("a");
+    let className = menuItem.innerText;
+    let classIndex = classesArr.findIndex((el) => {
+      if (el.name === className) {
+          return true;
+        }
+    });
+    title.innerText = classesArr[classIndex].name;
+    description.innerText = classesArr[classIndex].description;
+    classImg.setAttribute("width", "240px");
+    classImg.setAttribute("height", "auto");
+    classImg.setAttribute("src","./assets/illustrations/" + menuItem.innerText + ".jpg");
+    classImg.setAttribute("alt", classesArr[classIndex].name);
+    linkHome.innerText = "Повернутися на головну сторінку";
+}
```

Comment:
Don't update elements. Create or update the template and append content to the element.